src/app/components/activity/feed/post/AppActivityFeedPostContent.vue

```diff
@@ -0,0 +1,78 @@
+<script lang="ts" setup>
```

Comment:
Gotta rename and move this component file since it's no longer just used in activity feed.

AppPostContent?